src/locales/cy/pages.yml

```diff
@@ -112,7 +117,8 @@ rti-p60-statutory-adoption-pay:
 rti-p60-statutory-maternity-pay:
   title: Rhowch swm y ‘tâl mamolaeth statudol’ o'ch P60
   h1: Rhowch swm y ‘tâl mamolaeth statudol’ o'ch P60
-  inset:
+  inset: Defnyddiwch eich P60 o'r flwyddyn dreth {{ currentYearRangeStart }} i {{ currentYearRangeEnd }} ateb y cwestiwn hwn.
+  insetDates:
```

Comment:
could this be named better? eg. `insetSingleTaxYear` and `insetMultipleTaxYears`